assets/src/data/eventespresso/core/reducers/test/entities.js

```diff
@@ -2,11 +2,13 @@
  * External imports
  */
 import { EventFactory } from '@test/fixtures';
+import { fromJS, Map } from 'immutable';
+import { DEFAULT_CORE_STATE } from '@eventespresso/model';
 
 /**
  * Internal dependencies
  */
-import {
+import entitiesHandler, {
 	receiveEntity,
```

Comment:
is this correct?
is that var supposed to be before the brace?

Reply:
yes, its the default export.